zipline-kotlin-plugin/src/main/kotlin/app/cash/zipline/kotlin/OutboundBridgeRewriter.kt

```diff
@@ -64,18 +65,19 @@ import org.jetbrains.kotlin.name.Name
  * val helloService: EchoService = zipline.get(
  *   "helloService",
  *   object : OutboundBridge<EchoService>(EchoSerializersModule) {
- *     val serializer_0 = EchoSerializersModule.serializer<EchoRequest>()
- *     val serializer_1 = EchoSerializersModule.serializer<EchoResponse>()
- *     override fun create(outboundCallFactory: OutboundCall.Factory): EchoService {
+ *     override fun create(context: Context): EchoService {
+ *       val serializer_0 = context.serializersModule.serializer<EchoRequest>()
+ *       val serializer_1 = context.serializersModule.serializer<EchoResponse>()
  *       return object : EchoService {
  *         override fun echo(request: EchoRequest): EchoResponse {
- *           val outboundCall = outboundCallFactory.create("echo", 1)
+ *           val outboundCall = context.newCall("echo", 1)
  *           outboundCall.parameter<EchoRequest>(serializer_0, request)
  *           return outboundCall.invoke(serializer_1)
  *         }
  *       }
  *     }
- *   })
+ *   }
+ * )
```

Comment:
This summarizes the entire outbound side of the PR. I’ve renamed `OutboundCall.Factory` to `OutboundBridge.Context` and now get individual parameter serializers from that thing in `create()` rather than earlier.